src/cloud/project/project-conf-files_services.md

```diff
@@ -182,6 +184,141 @@ During the deployment process, {{site.data.var.ct}} checks installed service ver
 
 To maintain Magento store security, update installed software versions before they reach EOL.
 
+## Change service version
+
+You can upgrade the installed service version for compatibility with the Magento version deployed in your Cloud environment.
+
+You cannot downgrade the service version for an installed service directly. However, you can create a new service with the required version. See [Downgrade service version](#downgrade-version).
+
+Use the [Service versions](#service-versions) table to check service version compatibility by Magento version. Note that some service versions supported by {{ site.data.var.ee }} are not supported on {{ site.data.var.ece }}.
+
+{:.bs-callout-warning}
+You must submit a support ticket to change the service configuration on Pro Production and Staging environments.
+
+{:.bs-callout-info}
+If you change the Elasticsearch service version, check the Elasticsearch composer package for compatibility with the new version. See [Elasticsearch software compatibility]({{ site.baseurl}}/cloud/project/project-conf-files_services-elastic.html#elasticsearch-software-compatibility).
+
+### Upgrade installed service version
+
+You can upgrade the installed service version by updating the service configuration in the `services.yaml` file.
+
+1. Change the [`type`](#type) value for the service in the `.magento/services.yaml` file:
+
+   > Original service definition
+
+   ```yaml
+   mysql:
+       type: mysql:10.2
+       disk: 2048
+   ```
+
+   > Updated service definition
+
+   ```yaml
+   mysql:
+       type: mysql:10.3
+       disk: 2048
+   ```
+
+1. Add, commit, and push your code changes.
+
+   ```bash
+   git add -A
+   ```
+
+   ```bash
+   git commit -m "Upgrade MySQL from MariaDB 10.2 to 10.3."
+   ```
+
+   ```bash
+   git push origin <branch-name>
+   ```
+
+### Downgrade version
```

Comment:
I feel like we should add a callout here reminding users that this is a destructive process and will delete their existing database data.

Reply:
We have this information in the description of the service id field 
```
Renaming a service in the services.yaml file permanently removes the following:
 - The existing service before creating a new service with the new name you specify.
 - All existing data for the service is removed. We strongly recommend you snapshot your environment before you change the name of an existing service.
```
Probably we need to duplicate it

Reply:
I clarified the first bullet, and added a warning after step 1 of the "Rename" procedure. See most recent commits.